pkg/apis/contrail/v1alpha1/config_types.go

```diff
@@ -223,6 +222,21 @@ func (c *Config) InstanceConfiguration(request reconcile.Request,
 	apiServerSpaceSeparatedList = apiServerSpaceSeparatedList + ":" + strconv.Itoa(*configConfig.APIPort)
 	redisServerSpaceSeparatedList = strings.Join(podIPList, ":"+strconv.Itoa(*configConfig.RedisPort)+" ")
 	redisServerSpaceSeparatedList = redisServerSpaceSeparatedList + ":" + strconv.Itoa(*configConfig.RedisPort)
+	var cassandraEndpoint string
+	if cassandraNodesInformation.Endpoint == "" {
+		cassandraEndpointList := configtemplates.EndpointList(cassandraNodesInformation.ServerList, cassandraNodesInformation.Port)
+		cassandraEndpoint = configtemplates.JoinListWithSeparator(cassandraEndpointList, " ")
+	} else {
+		cassandraEndpoint = cassandraNodesInformation.Endpoint
+	}
```

Comment:
I think that if we simply store the ClusterIP in the ServerIPList it would simplify things.

Reply:
Done